bundler.d/test.rb

```diff
@@ -26,3 +21,5 @@
   gem 'robottelo_reporter', '~> 0.1'
   gem 'webmock'
 end
+
+gem 'theforeman-rubocop', '~> 0.0.3', groups: [:test, :rubocop]
```

Comment:
Please add the theforeman account to the:

https://rubygems.org/gems/theforeman-rubocop

This one:

https://rubygems.org/profiles/theforeman

Also I encourage you to start with 1.0.0 but this is just my opinion :-)

Reply:
Also link to homepage does not work (404): https://github.com/theforeman/theforeman-rubocop

Also the description could be little bit better:

> This gem guides and shares rubocop style configs.

Something like "Shared Rubocop configuration for theforeman.org family of projects" or something like that.

Reply:
I've added theforeman to gem owners, moved the repo to the theforeman org (so the homepage works now) and I will update the description :)

Reply:
I think semver suggests starting with `0.1.0` so you can easily push `0.1.1` without everyone having to update their gemfiles.

Reply:
I'd like the version `0.1.0` be clean core `.rubocop.yml`.
That is by no means work done as I belive there is many rubocop styles we want to enable/configure, but it would be a good start IMHO.
But yeah, before merging this PR we should release `0.1.0`.

Reply:
> I've added theforeman to gem owners

Still don't see these at https://rubygems.org/gems/theforeman-rubocop please do this we can't merge until we have full control of the rubygem. It bites!